core/client/src/main/java/alluxio/client/file/FileSystemUtils.java

```diff
@@ -51,6 +41,7 @@
   // prevent instantiation
   private FileSystemUtils() {}
 
+  // TODO(chaomin): remove this unused util.
```

Comment:
then should the class be annotated with `@PublicApi`? also, if we decide to keep this method, I suggest we convert its method to using `CommonUtils.waitFor`.